packages/frontend-v2/components/Error/ClientSideError.tsx

```diff
@@ -1,12 +1,56 @@
+import { Flex, Heading, Image, Text } from "@chakra-ui/react";
+import { GraduatePostAuthHeader } from "../Header";
+import { GraduateButtonLink } from "../Link";
+
 export const ClientSideError: React.FC = () => {
   return (
-    <>
-      <div>
-        <h1>GraduateNU</h1>
-      </div>
-      <div>
-        <p>There was an error rendering your page.</p>
-      </div>
-    </>
+    <Flex flexDirection="column" height="100vh" overflow="hidden">
+      <GraduatePostAuthHeader />
```

Comment:
Heres the bug I was talking about in the review. This is the header **after** you log in. We should only render this version after the user logs in. If its before, then it should be the regular header.